src/components/StoreSearch/StoreSearch.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { searchStores } from '../../actions/yelpAPI';
+
+export class StoreSearch extends React.Component {
+  getPosition(options) {
+    return new Promise(function(resolve, reject) {
+      navigator.geolocation.getCurrentPosition(resolve, reject, options);
+    });
+  }
+
+  convertDistance(meters) {
```

Comment:
In the future the conversion to display should happen in it's own component. This way it's clearer